src/hotspot/share/runtime/vmStructs.cpp

```diff
@@ -768,9 +768,6 @@ typedef HashtableEntry<InstanceKlass*, mtClass>  KlassHashtableEntry;
   /* JavaThread (NOTE: incomplete) */                                                                                                \
   /*********************************/                                                                                                \
                                                                                                                                      \
-  volatile_nonstatic_field(JavaThread,         _suspended,                                    bool)                                  \
```

Comment:
Have you checked whether any code in Serviceabiity Agent is depending on this field?

Reply:
I have looked but not find any, also the tests passes.